kitchen-sink-example.yaml

```diff
@@ -0,0 +1,2 @@
+# The file format version
+file_format: "0.1"
```

Comment:
@tsloughter one thing to note is if this is a string we have to double quote it (or use the `!!str` syntax) else it will be interpreted as a number.

Reply:
Or use 3 digits, 0.1.0, I think?

Reply:
This was discussed a bit [here](https://github.com/MrAlias/otel-schema/issues/15). Obviously everything is experimental and still subject to change. Can we merge as is and open an issue to discuss whether a patch version is needed?